### DISCUSSION TITLE
Welcome to OpenPDF Discussions!

### DISCUSSION THREAD

**Discussion by asturio** [LibrePDF/OpenPDF#761] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
  I think this is a better place, directly at GitHub instead of [Gitter](https://gitter.im/LibrePDF/OpenPDF).

**Comment by sebr72** [comment]
Hi, I am interested in OpenPDF and I would like to understand how the project is "driven" and see if I might be able to contribute. (I asked a question a couple of weeks ago, but I didn't get much reaction).

**Reply by asturio** [reply]
Sorry for that. The activity in OpenPDF varies a bit.

The project grew over the years and had during that time many contributors and maintainers. Sometimes more, and sometimes less.

Any contribution is welcome, from fixing a typo, adding documentation or translation, fixing bugs or adding fresh new features (like more PDF/A-support). Info about the "official" contributors are in [Contributors.md](https://github.com/LibrePDF/OpenPDF/blob/master/Contributors.md).

The maintainers were also in different period of time more or less active.

Take also a look at [CONTRIBUTING.md](https://github.com/LibrePDF/OpenPDF/blob/master/CONTRIBUTING.md)

Pull requests are very welcome for fixing bugs or adding new features.

If you want to contribute in a more regular basis, you may also become a maintainer. But this is only needed for some very special tasks like publishing new releases, or maybe be a fallback for other maintainers, if these are on a stressful time in their lives.

For my side, I must say, that I'm not very active in the discussions and prefer managing issues, and pull request. And creating new releases. And to be frank, in the last year there were not many activity from my side. So that's why any help is welcome, to keep the project running and the community alive.
